launcher/image/fixup_oem.sh

```diff
@@ -6,6 +6,16 @@ main() {
   fi
   mount /dev/sda12 /mnt/disks/efi
   sed -i -e 's|systemd.mask=usr-share-oem.mount||g' /mnt/disks/efi/efi/boot/grub.cfg
+
+  # TODO: Remove this fix once the upstream customizer fixed the bug.
```

Comment:
Is there an available reference to this?

Reply:
I don't think so, I can create one internally to track.